scan-command/src/main/java/io/ballerina/scan/utils/ScanUtils.java

```diff
@@ -276,6 +301,242 @@ private static void unzipReportResources(InputStream source, File target) throws
         }
     }
 
+    /**
+     * Returns the {@link ScanTomlFile} representation of a local/remote Scan.toml file.
+     *
+     * @param project      Ballerina project
+     * @param outputStream print stream
+     * @return an in-memory representation of the Scan.toml file
+     */
+    public static ScanTomlFile loadScanTomlConfigurations(Project project, PrintStream outputStream) {
+        Path targetDir = project.targetDir();
+
+        if (project.kind().equals(ProjectKind.BUILD_PROJECT)) {
+            if (project.currentPackage().ballerinaToml().isPresent()) {
+                BallerinaToml ballerinaToml = project.currentPackage().ballerinaToml().get();
+                TomlDocument ballerinaTomlDocument = ballerinaToml.tomlDocument();
+                Toml ballerinaTomlDocumentContent = ballerinaTomlDocument.toml();
+                Toml scanTable = ballerinaTomlDocumentContent.getTable(SCAN_TABLE).orElse(null);
+
+                if (scanTable != null) {
+                    TomlValueNode configPath = scanTable.get(SCAN_FILE_FIELD).orElse(null);
+
+                    if (configPath != null) {
+                        String scanTomlPath = configPath.toNativeValue().toString();
+                        Optional<Path> path = convertConfigurationPath(scanTomlPath);
+                        Optional<Path> absPath = path.isEmpty() || path.get().isAbsolute() ? path
+                                : Optional.of(project.sourceRoot().resolve(scanTomlPath));
+
+                        if (isLocalFile(absPath)) {
+                            return loadScanFile(targetDir, absPath.get(), outputStream);
+                        } else {
```

Comment:
Since we are returning from the above else block, we can remove the `else` part.

``` java
if (isLocalFile(absPath)) {
    return loadScanFile(targetDir, absPath.get(), outputStream);
}
try {
    URL url = new URL(scanTomlPath);
    return loadRemoteScanFile(targetDir, url, outputStream);
} catch (MalformedURLException ex) {
    throw new RuntimeException("Failed to retrieve remote configuration file " +
                                        "with exception: " + ex.getMessage());
}